generators/search-map.js

```diff
@@ -21,14 +24,17 @@ var fs = require('fs'),
 module.exports = function(docMap, siteConfig) {
 	var searchMap = {},
 		name;
-
 	for (name in docMap) {
 		if (docMap.hasOwnProperty(name)) {
 			var docObj = docMap[name];
+			var description = markdownParser.parse(docObj.description);
+			description = markdownRenderer.render(description);
+			description = striptags(description);
```

Comment:
No technical reason I can think of, especially if https://github.com/canjs/bit-docs-html-canjs/pull/266 is acceptable. Will double check with @mickmcgrath13 as he originally implemented it so perhaps he knows something I don't.